src/javascript/EditPanel/EditPanelContent/FormBuilder/Section/FieldSet/Field/SelectorTypes/Category/Category.jsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import * as PropTypes from 'prop-types';
+import {FieldPropTypes} from '~/EditPanel/FormDefinitions/FormData.proptypes';
+import {FastField} from 'formik';
+import {DropdownTreeSelect} from '~/DesignSystem/DropdownTreeSelect';
+
+const data = [
```

Comment:
this data should stay here?

Reply:
Not at all BACKLOG-11737 will remove this code :)